SU-HSDO/suhumsci
# Shs 26 accordion utility class

## Summary
This adds a new utility class for accordions so that the appearance can be modified. 

## Steps to Test
1. Go to the Accordions page (usually `/accordions`). Edit the Layout (tab at top of page). Go to the Main Region and click Configure (screenshot below). In the 'Field Formatter Class' text box, add `hb-accordion-secondary`. Save the layout.
2. Visit the Accordion page and note the appearance is now changed. 
3. The look should be changed according to the ticket:
- In the sitting state, the outline is the secondary color with the standard body text color for heading/icon. 
- Hover animates to fill the box solid with primary color background and white heading/icon.
- On click, it returns to heading/icon in text color, but outline is now primary. Primary outline wraps around the entire content block.
- On hover of heading/icon block in the expanded state, heading/icon switches to secondary color.
4. Verify the same for the Traditional theme
5. Verify the new utility class didn't alter anything else except the accordion on the page

## Notes
I don't see a release branch anywhere, so I'm leaving it on develop for now. This PR only has one commit. 

## Screenshots
![Screen Shot 2022-09-13 at 8 44 30 AM](https://user-images.githubusercontent.com/8405274/189904809-61968ecf-b396-4271-90da-b631cb6e3f88.png)

![Screen Shot 2022-09-13 at 8 41 37 AM](https://user-images.githubusercontent.com/8405274/189904861-96a0f3d4-43a9-40a4-8b63-fb770350a5dd.png)
